backend/spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -168,14 +167,14 @@ def user
     context "when the user cannot manage roles" do
       it "cannot set roles" do
         expect {
-          put :update, params: { id: user.id, user: { first_name: "Bob", spree_role_ids: [dummy_role.id] } }
+          put :update, params: { id: user.id, user: { name: "Bob", spree_role_ids: [dummy_role.id] } }
         }.not_to change { user.reload.spree_roles.to_a }
       end
 
       it "cannot clear roles" do
         user.spree_roles << dummy_role
         expect {
-          put :update, params: { id: user.id, user: { first_name: "Bob" } }
+          put :update, params: { id: user.id, user: { name: "Bob" } }
```

Comment:
Even here, I'd change it to look like a real full name.

Reply:
👍